checks/checks.yaml

```diff
@@ -51,6 +51,14 @@ checks:
       - >-
         Signup for automatic dependency updates with dependabot or renovatebot and place the config
         file in the locations that are recommended by these tools.
+  Binary-Artifacts:
+    description: >-
+      This check tries to determine if a project has binary artifacts in the source repository.
+      The files with following extensions are checked ".o", ".so", ".ko", ".elf", ".exe", ".dll", ".deb", 
+      ".rpm", ".jar", ".pyc". These binaries could be compromised artifacts.
```

Comment:
we can also add that binaries don't provide provenance information and are hard to patch when vulnerabilities are discovered - since we don't know what they contain.